### ISSUE TITLE
Enhancing documentation by adding Screenshots

### ISSUE THREAD

**Issue by AbdulMalikDev** [CCExtractor/rutorrent-flutter#138]
## Proposed Addition/Enhancement

Addition of Screenshots (all existing 10 screens) to the documentation to give an inside peak as well as serve the purpose of introduction to the application.

**Comment by tusharojha** [comment]
I think this would be a nice add when the app is published on stores. Till then I don't think this is needed.

Ps. It's just my opinion. So, @harchani-ritik is the right person to answer.

**Comment by nb9960** [comment]
It would be better to add gifs (or maybe use arrows in screenshots) describing usage.

**Comment by harchani-ritik** [comment]
Yes, I agree will all the suggestions, screenshots aren't required here in the README (can be included when we publish the app) and GIF's will be better to serve the purpose of introduction to the new users (Not necessarily for every functionality but primarily for Login).
